pkg/view/component/base.go

```diff
@@ -13,6 +13,7 @@ const (
 	typeCodeBlock           = "codeBlock"
 	typeContainers          = "containers"
 	typeDonutChart          = "donutChart"
+	typeEditor              = "editor"
```

Comment:
This can be done in a separate PR, but also needs an entry in `unmarshal.go` xr https://github.com/vmware-tanzu/octant/issues/646